docs/manual/scala/guide/production/ProductionOverview.md

```diff
@@ -35,11 +35,11 @@ The production environment determines the methods for packaging your services, m
 
     * If you don't use a service orchestration platform and can determine the addresses of some of your nodes in advance of deploying them, Akka Cluster can be configured manually by listing the addresses of [seed nodes](https://doc.akka.io/docs/akka/current/cluster-usage.html#joining-to-seed-nodes) in the service configuration.
 
-    * When you don't specify a static list of `seed-nodes`, Lagom will enable the [Akka Cluster Bootstrap](https://developer.lightbend.com/docs/akka-management/current/bootstrap/index.html) extension for integration with other service discovery infrastructure. Using Akka Cluster Bootstrap requires setting up a `Service Discovery` method (for example [Service Discovery for Kubernetes](https://developer.lightbend.com/docs/akka-management/current/discovery/kubernetes.html)). See the [[Lagom Cluster|Cluster]] documentation for more information.
+    * When you don't specify a static list of `seed-nodes`, Lagom will enable the [Akka Cluster Bootstrap](https://doc.akka.io/docs/akka-management/current/bootstrap/index.html) extension for integration with other service discovery infrastructure. Using Akka Cluster Bootstrap requires setting up a `Service Discovery` method (for example [Service Discovery for Kubernetes](https://doc.akka.io/docs/akka-management/current/discovery/kubernetes.html)). See the [[Lagom Cluster|Cluster]] documentation for more information.
```

Comment:
Is it intended to keep `current` instead of `1.0` in this page?

Reply:
I left it just as ambiguous as before, without validating each.

Reply:
Alright.